lib/inbox.dart

```diff
@@ -10,9 +10,8 @@ class Inbox extends StatelessWidget
     Inbox( { super.key } );
     static const String route = '/inbox';
 
-    bool inboxEmpty = true;
-
-    //Get the current signed in user
+    final CollectionReference inboxRef = FirebaseFirestore.instance
+                                                       .collection('rrdbInbox');
     User? currUser = FirebaseAuth.instance.currentUser;
 
     String? returnEmail()
```

Comment:
Is the returnEmail function necessary? We already have a reference to currUser from above.